lib/protocol/events.js

```diff
@@ -227,5 +227,16 @@ function eventsProtocolFactory (Constants, messenger, assert) {
         );
     };
 
+    EventsProtocol.prototype.publishNodeAlert = function (nodeId, data) {
+        assert.string(nodeId);
+        assert.object(data);
+
+        return messenger.publish(
```

Comment:
It's not limited to a task, removing a node by API will also could use it. and node alert itself also don't have relationship with task.